src/sentry/models/group.py

```diff
@@ -401,6 +407,14 @@ class Group(Model):
     short_id = BoundedBigIntegerField(null=True)
 
     objects = GroupManager(cache_fields=("id",))
```

Comment:
Nit: By convention we usually make sure the manager is declared after the field declarations